Readme.md

```diff
@@ -1,26 +1,47 @@
 Slack app for Voting on Stories and Tickets remotely and asynchronously.
 
+## Usage
+
+1. Install this app to your Slack team visiting http://asyncvoter.tk and pressing on the `Add to Slack` button.
```

Comment:
@arreche Can we explicitly mention in a step that the contributor has to create a new slack team? 'To your slack team' isn't too clear IMO